workflow/Snakefile

```diff
@@ -0,0 +1,140 @@
+import pandas as pd
+
+configfile: "/storage/scratch01/groups/bu/myeloma_altum/agostoli/board_alegosto01/myelong/config/config.yaml"
```

Comment:
I just realised you have a lot of hardcoded full paths like this one in the code. It's very important to change them to relative paths so anyone can run the code without modifications. This one should only be `config/config.yaml`